src/screens/CreatePost/index.js

```diff
@@ -790,7 +770,7 @@
             topic={listTopic.length > 0}
             listTopic={renderListTopic()}
             label="Add Communities"
-            labelStyle={styles.hastagText}
+            labelStyle={styles.hashtagText}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `labelStyle` prop is now using `styles.hashtagText` instead of `styles.hastagText`. Ensure that this change was intentional and that the new style exists and is correctly applied.
```diff
+ labelStyle={styles.hashtagText}
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->